### ISSUE TITLE
Transform the docs folder in this repo into a handbook site for OL Platform Engineering

### ISSUE THREAD

**Issue by feoh** [mitodl/ol-infrastructure#2008]
# Description/Context

Right now, all our docs live at https://github.com/mitodl/ol-infrastructure/tree/main/docs are just Markdown files.

Navigating to a given doc is moderately painful, and there's no searchability short of good old fashioned grep in the checked out repo.

This isn't amazing and isn't the kind of easy consumability we want in order to bar raise our operational excellence.

# Plan/Design

Use the example set by http://mitodl.github.io/handbook/ to create a separate MIT OL Platform Engineering Github Pages/Jekyll site.

Organize it by categories at first, with a very brief routing page to help people understand which category will contain the docs they want.

Ensure that the site is searchable from the webpage.

**Comment by feoh** [comment]
I think using the Hugo Book theme - https://themes.gohugo.io/themes/hugo-book/ will simplify this a bunch. I already have a branch with a decent looking site up. Just need to figure out how the inline directives will create the menu structure we want.

**Comment by feoh** [comment]
After days of churn where I couldn't get a theme that worked for our needs and also couldn't get past Hugo's insane complexity and deploying that to Github Pages, I've dropped back and punted to Nikola, the simple Python SSG I use for my own blogs.

I was able to get an ugly site up in minutes.

Now I need to:

- Configure the *built in* Github pages deployment functionality
- Make the site less ugly. Like, right now it's unusably ugly :) You just get a hamburger in the top left, which you click to show a Pages link, click that and you get our document categories, under which are all our documents.
- Figure out why the documents show up as both .md and .html
